projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html

```diff
@@ -1,13 +1,13 @@
 <ng-container *ngIf="facetList$ | async as facetList">
-  <h4 *ngIf="facetList?.activeFacets?.length > 0">
+  <h4 *ngIf="facetList.activeFacets && facetList.activeFacets.length > 0">
```

Comment:
Still not sure about operator precedence between && and >.
Do you think it could be an issue? Perhaps we need to try it out.

Reply:
I will add `()` if we are not sure.